index.js

```diff
@@ -98,6 +99,54 @@ app.post("/github_discord_urls", async (req, res) => {
   }
 });
 
+async function get_auth_user_info_by_email(email) {
+  const data = await pool.query(
+    `select * from auth_user_info where email = $1`,
+    [email]
+  );
+  if (data.rows.length <= 0) {
```

Comment:
3 conditions here. 
1. if length == 0, return null 
2. if length == 1, return first row
3. if length > 1, throw a 500 error (since email field should be unique in this case)